packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -188,7 +208,7 @@ export function DevtoolsView(): React.ReactElement {
 		return (): void => {
 			messageRelay.off("message", messageHandler);
 		};
-	}, [messageRelay, setSupportedFeatures]);
+	}, [messageRelay, setSupportedFeatures, topLevelLogger, telemetryOptInLogger]);
```

Comment:
Once you make the change we discussed in Teams, I think `topLevelLogger` will not be required here